client/lib/ui/NotificationListItem.js

```diff
@@ -52,7 +52,7 @@ export default React.createClass({
 
     return (
       <FastButton component="div" className={classNames('notification', this.props.kind, {'seen': message.get('_seen')})} onClick={ev => this.props.onClick(ev, this.props.nodeId)}>
-        <MessageText className="title" content={message.get('content')} maxLength={140} />
+        <MessageText className="title" content={(message.get('content') || '').trim()} maxLength={140} />
```

Comment:
I am not sure how the code gets at all to rendering a message without text; since the exceptions were complaining about undefined values, I've added a guard for that.
The [immutable.js docs](https://facebook.github.io/immutable-js/docs/#/Map/get) tell that it possible, but also that

> [...] it is possible a key may be associated with an undefined value, so if notSetValue is not provided and this method returns undefined, that does not guarantee the key was not found.

...so that the case of the message actually having an `undefined` content is not covered.
¯\_(ツ)_/¯